endpoints-control/src/test/java/com/google/api/control/model/KnownMetricsTest.java

```diff
@@ -92,48 +84,52 @@ public void shouldUpdateOperationCorrectly() {
               .setMetricName(KnownMetrics.PRODUCER_REQUEST_COUNT.getName())
               .addMetricValues(MetricValue.newBuilder().setInt64Value(1L))
               .build()),
-      new StructuredTest(KnownMetrics.PRODUCER_BY_CONSUMER_REQUEST_COUNT,
-          MetricValueSet
-              .newBuilder()
-              .setMetricName(KnownMetrics.PRODUCER_BY_CONSUMER_REQUEST_COUNT.getName())
-              .addMetricValues(MetricValue.newBuilder().setInt64Value(1L))
-              .build()),
       new StructuredTest(KnownMetrics.CONSUMER_REQUEST_ERROR_COUNT),
-      new StructuredTest(new ReportRequestInfo().setResponseCode(400),
+      new StructuredTest(
+          (ReportRequestInfo) new ReportRequestInfo()
```

Comment:
All of the casts are because the *ApiKey methods are in OperationInfo and as such return OperationInfo rather than ReportRequestInfo.